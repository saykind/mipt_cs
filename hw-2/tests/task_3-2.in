10
.1 .01 .001 .0001 .00001 .000001 .0000001 .00000001 .000000001 .0000000001
