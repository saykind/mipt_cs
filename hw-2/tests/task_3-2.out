0.1 0.01 0.001 0.0001 1e-05 1e-06 1e-07 -nan -nan -nan 